Multisig/App/RemoteNotificationHandler.swift

```diff
@@ -225,13 +264,24 @@ class RemoteNotificationHandler {
                 // The signing might fail in case the underlying key store is not available. In this case
                 // we abort the whole process. The registration will be attempted again on the next
                 // app coming to foreground event.
-                let signResult = try Self.sign(safes: safes,
-                                               deviceID: deviceID,
-                                               token: pushToken,
-                                               timestamp: timestamp)
-                
-                safeRegistration.append(SafeRegistration(chainId: chainSafes.chain.id!, safes: safes,
-                                                         signatures: signResult.signatures))
+
+                if AppConfiguration.FeatureToggles.securityCenter {
```

Comment:
When testing this PR on my phone the app crashes here (in AppConfiguration.FeatureToggles.securityCenter). Regardless whether security center is enabled or not.

Reply:
unable to reproduce

Reply:
Will have another look at the latest version.

Reply:
I can still reproduce the crash as follows:
- delete dev app
- install dev app
- add safe
- add owner for safe 

App will crash in "enter key name" screen
App will crash on restart after hitting "skip" on the update screen